mindmeld/active_learning/classifiers.py

```diff
@@ -235,17 +337,31 @@ def train(self, data_bucket: DataBucket, heuristic: Heuristic):
         """
         eval_stats = defaultdict(dict)
         eval_stats["num_sampled"] = len(data_bucket.sampled_queries)
-        confidences_2d = self.train_single(data_bucket, eval_stats)
+        confidences_2d, entity_confidences, eval_stats = self.train_single(
+            data_bucket, heuristic, eval_stats
+        )
         return_confidences_3d = isinstance(heuristic, MULTI_MODEL_HEURISTICS)
-        confidences_3d = (
-            self.train_multi(data_bucket) if return_confidences_3d else None
+        confidences_3d, entity_confidences_3d = (
```

Comment:
We should probably make this change for when we use the 3d confidences. Seems like we're not using it at the moment.